protos/processor.proto

```diff
@@ -26,7 +26,21 @@ import "transaction.proto";
 //
 // The protocol_version field is used to check if the validator supports
 // requested features by a transaction processor.
+// Following are the versions supported:
```

Comment:
This comment says the rfc is at text/0000-raw-txn-header.md  but it actually been moved to text/0023-raw-txn-header.md Please either update this or remove that line and just point to the pr

Reply:
This also should be updated in the other sdk prs as well,

Reply:
Ok, I will consider Python SDK > Rust SDK > CXX SDK one at a time. Because Python SDK has related code changes to pass test cases.

Reply:
This is fixed, the latest commit message does not mention the RFC number.